app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+class ReviewsController < ApplicationController
+  before_action :find_book
+
+  def create
+    @review = @book.reviews.create(review_params)
+
+    redirect_to @book
+  rescue StandardError => e
+    Rails.logger.error(e.message)
+  end
+
+  def edit
+    @review = Review.find(params[:id])
```

Comment:
We should better move this to `before_action` since it is repeatedly called in several methods and it should be

```ruby
@book.reviews.find(params[:id])
```

to scope down only the reviews that belong to the book